airflow/contrib/operators/app_engine_operator.py

```diff
@@ -83,15 +84,26 @@ class AppEngineOperatorSync(BaseOperator):
     AppEngineOperatorSync calls an API endpoint in App Engine and waits for a response. If the response has a 4xx or 5xx
     status, the task is considered failed. If a body is included in the response, it will be stored in the
     `return_value` XCom. JSON and YAML will be deserialized automatically.
+
+    :param task_id: Name of the task to appear in Airflow UI
+    :type task_id: str
+    :param command_name: Full name of the App Engine command to be called (e.g. engine.core.commands.ExCommand)
+    :type command_name: str
+    :param command_params: Named parameters of App Engine command to be called
+    :type command_params: dict
+    :param http_conn_id: ID mapped to the host to which the call is sent. Can be defined in Airflow UI
+    :type http_conn_id: str
+    :param kwargs: Named parameters to pass to BaseOperator constructor
+    :type kwargs: dict
     """
     template_fields = ('command_name', 'command_params', 'job_id',)
 
     @apply_defaults
     def __init__(self,
                  task_id,
                  command_name,
-                 command_params,
-                 http_conn_id='http_default',
+                 command_params={},
```

Comment:
we should try to avoid mutable defaults: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

in the __init__, you can put something like:

if not command_params:
    command_params = {}

Reply:
this is an object! I don't know how to read! well done!

Reply:
Not an issue for object instantiation